autonomy/cli/deploy.py

```diff
@@ -272,6 +280,26 @@ def run_deployment_from_token(  # pylint: disable=too-many-arguments, too-many-l
     run_deployment(build_dir)
 
 
+def update_multisig_address(service_path: Path, address: str) -> None:
+    """Update the multisig address on the service config."""
+
+    with open_file(service_path / DEFAULT_SERVICE_FILE) as fp:
+        config, *overrides = yaml_load_all(
+            fp,
+        )
+
+    for override in overrides:
+        if override["type"] == SKILL:
+            addresses = override["setup_args"]["args"]["setup"]["safe_contract_address"]
+            override["setup_args"]["args"]["setup"]["safe_contract_address"] = [
+                *addresses,
+                address,
```

Comment:
There's only one possible address, so I would not append the list.

Reply:
So why keep it as the list? can't we declare it as a string?